### PR TITLE
Use server's default timezone as default timezone

### PR DESCRIPTION
**Pull request by qwerty287**
Use the default timezone of the server instead of UTC as default timezone.

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -78,7 +78,7 @@
 	|
 	*/
 
-	'timezone' => env('TIMEZONE', 'UTC'),
+	'timezone' => env('TIMEZONE', date('T')),
```
Comment (nagmat84) [inline config/app.php]:
```suggestion
	'timezone' => env('TIMEZONE', date('e')),
```

We need the timezone identifier or a so-called level-3 timezone, e.g. `Europe/Paris`. A timezone abbreviation like "CEST" (level-2) or a simple time offset like "+02:00" does not suffice.

File: .env.example [hunk 2]
```diff
@@ -29,7 +29,8 @@ DB_USERNAME=
 DB_PASSWORD=
 DB_LOG_SQL=false
 
-TIMEZONE=UTC
+# Application timezone. If not specified, the server's default timezone is used.
+# TIMEZONE=UTC
```
Comment (nagmat84) [inline .env.example]:
```suggestion
# Application timezone. If not specified, the server's default timezone is used.
# Requires a named timezone identifier.
# See https://www.php.net/manual/en/timezones.php for the list of supported timezones,
# except for those listed below "Others".
# Don't use a timezone offset (like +01:00) or a timezone abbreviation (like CEST)
# TIMEZONE=Europe/Paris
```

**Review by nagmat84 — APPROVED** [review]
LGTM, but see proposed changes.

I don't see any drawback or problem in using the server's timezone as a default, but maybe @kamil4 or @ildyria. So I invited them for a review.

File: config/app.php (hunk 1 above)
Comment (qwerty287) [reply config/app.php]: But wasn't the default before just `UTC` which is the level-2 variant?

File: .env.example (hunk 2 above)
Comment (qwerty287) [reply .env.example]: Should we use `Europe/Paris` as default timezone?

Comment (qwerty287) [reply .env.example]: Actually that's just an example so it's not important

File: config/app.php (hunk 1 above)
Comment (nagmat84) [reply config/app.php]: Yes and it produced a warning in the diagnostics to change that.

File: .env.example [hunk 3]
```diff
@@ -29,7 +29,12 @@ DB_USERNAME=
 DB_PASSWORD=
 DB_LOG_SQL=false
 
-TIMEZONE=UTC
+# Application timezone. If not specified, the server's default timezone is used.
+# Requires a named timezone identifier.
+# See https://www.php.net/manual/en/timezones.php for the list of supported timezones,
+# except for those listed below "Others".
```
Comment (d7415) [inline .env.example]:
```suggestion
# See https://www.php.net/manual/en/timezones.php for the list of supported timezones.
```

The list itself advises against these, so I don't see that we have to. Additionally, UTC remains a valid option.

**Review by kamil4 — APPROVED** [review]
I'm OK with that.

**Pull request merged**